docs/docs/events.md

```diff
@@ -5,21 +5,41 @@
 Called once the app is launched. This event is used to set the Application's initial layout - after which the app is ready for user interaction.
 
 ```js
-Navigation.events().registerAppLaunchedListener(() => {
+appLaunchedListener = Navigation.events().registerAppLaunchedListener(() => {
```

Comment:
Please add `const` in front of these variables 👍 Same thing for example in line 53